### DISCUSSION TITLE
Features to make part of the player's modeling invisible from a first-person perspective

### DISCUSSION THREAD

**Discussion by applelike2** [xTracr/RealCamera#80] [Ideas]
![2024-06-02_23 23 30](https://github.com/xTracr/RealCamera/assets/155553984/e6035f2f-6431-4f9c-9992-b0c040f7275a)
I use yes steve model mod but part of the hair modeling obscures the screen.
Is it possible to add a feature that makes part of the player's modeling invisible in first person?

**Comment by xTracr** [comment]
![screenshot_2024_6_2_22_42](https://github.com/xTracr/RealCamera/assets/57320980/78c246e8-34aa-4979-89de-780ee907870b)
Increasing this value may help you.

**Reply by applelike2** [reply]
thanks you!
